test/Makefile

```diff
@@ -0,0 +1,64 @@
+LIBDIR=../lib
+GTEST_MODULE=$(LIBDIR)/googletest
+GTEST_DIR=$(LIBDIR)/googletest/googletest
+GTEST_SOURCEDIR=$(GTEST_DIR)/src
+GTEST_HEADERS=$(GTEST_DIR)/include/gtest/*.h \
+              $(GTEST_DIR)/include/gtest/internal/*.h
+GTEST_SRCS_=$(GTEST_SOURCEDIR)/*.cc $(GTEST_SOURCEDIR)/*.h $(GTEST_HEADERS)
+
+CPP_NETLIB_DIR=$(LIBDIR)/cpp-netlib
```

Comment:
We should put these in a common location and include them in both Makefiles.

Reply:
Ack. Perhaps I can handle this in a separate PR, so that the code changes are restricted to the test directory.

Reply:
Ok, let's add a TODO, please?